sdcm/mgmt/cli.py

```diff
@@ -436,17 +436,8 @@ def create_backup_task(self, dc_list=None,  # pylint: disable=too-many-arguments
         if legacy_args:
             cmd += f" {legacy_args}"
         res = self.sctool.run(cmd=cmd, parse_table_res=False)
-        if not res.stdout:
-            raise ScyllaManagerError("Unknown failure for sctool '{}' command".format(cmd))
-
-        if res.stderr:
-            # We are ignoring "unable to resolve host" messages as they are being show on every command, because
-            # we had issues changing Ubuntu machines hostname.
-            if res.stderr.count('unable to resolve host'):
-                pass
-            else:
-                LOGGER.debug("Encountered an error on '{}' command response".format(cmd))
-                raise ScyllaManagerError(res.stderr)
+        if not res.ok:
```

Comment:
Remove these two lines too, sctool.run already raising the error on non-zero status:
https://github.com/scylladb/scylla-cluster-tests/blob/bbbc9fc6064912d38def66a712ddba2056267e3b/sdcm/mgmt/cli.py#L1013-L1017

Reply:
well the scope of this PR is only for scylla-cloud-manager to be able to operate as fast as possible.
i prefer deferring this suggestion to alter PR so we'll focus testing it on standard manager (not scylla-cloud-manager)

Reply:
These two lines are never gonna be executed since sctool.run will raise exact same exception in such case.
So, please remove them too, and let's merge it